AppTemplate/AppTemplate/Renderer.swift

```diff
@@ -412,6 +425,20 @@ class Renderer: NSObject {
         }
     }
 
+    static func applySwitchFacts(switch_: UISwitch, facts: Json) {
+        for key in facts.keys {
+            switch key {
+            case "checked":
+                if let value = facts[key] as? Bool {
+                    switch_.isOn = value
+                }
```

Comment:
Obviously no facility for this yet, but `checked` is kind of a required attribute here. Have you thought about how to possibly either warn at runtime or perhaps error out at compile time here?

Or might it make sense to abandon the 'list of attributes' interface in this case? So `switch` would have to take a value?

```elm
switch : Bool -> List (Attribute msg) -> Element msg
```